README.md

```diff
@@ -24,43 +24,150 @@ To keep the code base and the API simple, zerolog focuses on JSON logging only.
 * `net/http` helpers
 * Pretty logging for development
 
-## Usage
-
+## Installation
 ```go
-import "github.com/rs/zerolog/log"
+go get -u github.com/rs/zerolog/log
 ```
+## Getting Started
+### Simple Logging Example
+For simple logging, import the global logger package **github.com/rs/zerolog/log**
+```go
+package main
 
-### A global logger can be use for simple logging
+import (
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
 
-```go
-log.Print("hello world")
+func main() {
+	// UNIX Time is faster and smaller than most timestamps
+	// If you set zerolog.TimeFieldFormat to an empty string,
+	// logs will write with UNIX time
+	zerolog.TimeFieldFormat = ""
+
+	log.Print("hello world")
 
-// Output: {"level":"debug","time":1494567715,"message":"hello world"}
+}
+
+// Output: {"time":1516134303,"level":"debug","message":"hello world"}
 ```
+> Note: The default log level for `log.Print` is *debug*
+----
+### Leveled Logging
 
+#### Simple Leveled Logging Example
 
 ```go
-log.Info().Msg("hello world")
+package main
+
+import (
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
 
-// Output: {"level":"info","time":1494567715,"message":"hello world"}
+func main() {
+	zerolog.TimeFieldFormat = ""
+
+	log.Info().Msg("hello world")
+
+}
+
+// Output: {"time":1516134303,"level":"info","message":"hello world"}
 ```
 
-NOTE: To import the global logger, import the `log` subpackage `github.com/rs/zerolog/log`.
+**zerolog** allows for logging at the following levels (from highest to lowest):
+- panic (`zerolog.PanicLevel`, 5)
+- fatal (`zerolog.FatalLevel`, 4)
+- error (`zerolog.ErrorLevel`, 3)
+- warn (`zerolog.WarnLevel`, 2)
+- info (`zerolog.InfoLevel`, 1)
+- debug (`zerolog.DebugLevel`, 0)
 
+You can set the Global logging level to any of these options using the `SetGlobalLevel` function in the zerolog package, passing in one of the given constants above, e.g. `zerolog.InfoLevel` would be the "info" level.  Whichever level is chosen, all logs with a level greater than or equal to that level will be written. To turn off logging entirely, pass the `zerolog.Disabled` constant.
+
+#### Setting Global Log Level
+This example uses command-line flags to demonstrate various outputs depending on the chosen log level.
 ```go
-log.Fatal().
-    Err(err).
-    Str("service", service).
-    Msgf("Cannot start %s", service)
+import (
+	"flag"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func main() {
+	debug := flag.Bool("debug", false, "sets log level to debug")
+
+	flag.Parse()
+	zerolog.TimeFieldFormat = ""
+
+	// Determine log level based on command line flag
+	// For this example, we set level to Info if Debug flag is not present
+	var lvl zerolog.Level
+	if *debug {
+		lvl = zerolog.DebugLevel
+	} else {
+		lvl = zerolog.InfoLevel
+	}
+
+	zerolog.SetGlobalLevel(lvl)
```

Comment:
Cosmetic comment: I would have written this that way to better show what's default:

```go
zerolog.SetGlobalLevel(zerolog.InfoLevel)
if *debug {
    zerolog.SetGlobalLevel(zerolog.DebugLevel)
}
```

Reply:
You're totally right - changed and committed!